### PR TITLE
feat(PP-38): responsive navbar

### PR DESCRIPTION
**Pull request by DmitrijSidorkin**
Added navbar layouts for tablet and mobile. Did not add icons for navbar mobile links that are displayed when no logged in because it did not seem necessary.

### CODE REVIEW

File: views/partials/navbar.ejs [hunk 1]
```diff
@@ -3,12 +3,29 @@
...
-    <a href="/account">My Account</a>
-    <a href="/logout">Log out</a>
+    <a class="nav-link" href="/play">Play</a>
+    <a class="nav-link" href="/account">My Account</a>
+    <a class="nav-link" href="/logout">Log out</a>
+
+    <a class="icon-wrapper" href="/play"
+      ><img
+        class="nav-icon"
+        src="https://res.cloudinary.com/dyguovdbc/image/upload/v1685537009/pixelify_assets/play_icon_q7dom6.png"
```
Comment (elnino97) [inline views/partials/navbar.ejs:16]: Those images you can store in the public folder and access them using relative paths, for example: `src="./play_icon.png"`

Comment (DmitrijSidorkin) [reply views/partials/navbar.ejs:16]: Good point, thanks!

File: views/partials/navbar.ejs [hunk 2]
```diff
@@ -3,12 +3,23 @@
...
+    <a class="icon-wrapper" href="/play"
+      ><img class="nav-icon" src="../icons/play_icon.png" alt="play"
+    /></a>
+    <a class="icon-wrapper" href="/account"
+      ><img class="nav-icon" src="../icons/account_icon.png" alt="account"
+    /></a>
+    <a class="icon-wrapper" href="/logout"
+      ><img class="nav-icon" src="../icons/logout_icon.png" alt="logout"
+    /></a>
+    <div class="icon-wrapper"></div>
```
Comment (elnino97) [inline views/partials/navbar.ejs:22]: What is the purpose of this empty `div`?

File: public/stylesheets/navbar.css [hunk 3]
```diff
@@ -17,15 +20,53 @@
...
 }
+
+.icon-wrapper {
+  display: none;
+}
+
+.nav-icon {
+  max-height: 30px;
+  align-self: center;
+  padding: 0 0;
```
Comment (elnino97) [inline public/stylesheets/navbar.css:45]: `padding: 0`

File: views/partials/navbar.ejs [hunk 4]
```diff
@@ -3,12 +3,23 @@
...
+    <a class="nav-link-guest" href="/register">Register</a>
     <% } else { %>
-    <a href="/play">Play</a>
-    <a href="/account">My Account</a>
-    <a href="/logout">Log out</a>
+    <a class="nav-link" href="/play">Play</a>
+    <a class="nav-link" href="/account">My Account</a>
+    <a class="nav-link" href="/logout">Log out</a>
+
+    <a class="icon-wrapper" href="/play"
```
Comment (elnino97) [inline views/partials/navbar.ejs:13]: Why not have these icons elements inside `a` elements above?

File: views/partials/navbar.ejs (hunk 2 above)
Comment (DmitrijSidorkin) [reply views/partials/navbar.ejs:22]: Is unintentional leftover code, will get rid of it

File: views/partials/navbar.ejs (hunk 4 above)
Comment (DmitrijSidorkin) [reply views/partials/navbar.ejs:13]: Thats a good point, didnt realize this while doing it. Will change it, thanks!

**Review by elnino97 — APPROVED** [review]

**Pull request merged**